client/client-multi/client-c/src/kaa/platform/ext_key_utils.h

```diff
@@ -82,7 +82,7 @@ kaa_error_t ext_encrypt_data(const uint8_t *input, size_t input_size, uint8_t *o
  * @param[out] output_payload_size  The size of the payload data.
  * @param[out] output               The buffer which contains dectypted data will be filled in.
  *
- * @return error code.
+ * @retval KAA_ERR_NONE on success. The other value means failure.
```

Comment:
It should go like:
```
@return Error code.
@retval KAA_ERR_NONE Success.
```

Actually, it's much better to document some return values, as the current documentation is true for all functions returning `kaa_error_t` -- that doesn't add much value but rather worsens things.